bin/cml/runner.js

```diff
@@ -470,6 +473,12 @@ exports.builder = (yargs) =>
         description:
           'Instance type. Choices: [m, l, xl]. Also supports native types like i.e. t2.micro'
       },
+      cloudPermissionSet: {
+        type: 'string',
+        default: '',
+        description:
```

Comment:
@DavidGOrtega should this be hidden?

Reply:
@dacbd sorry, what do you mean by hidden?

Reply:
Yargs has a hidden attr you can use

Reply:
Ah! I see. I do not see any reason to hide it. Indeed is a new interesting security feature